themes/default/content/registry/packages/time/installation-configuration.md

```diff
@@ -10,7 +10,6 @@ The Pulumi Time provider is available as a package in all Pulumi languages:
 
 * JavaScript/TypeScript: [`@pulumiverse/time`](https://www.npmjs.com/package/@pulumiverse/time)
 * Python: [`pulumiverse_time`](https://pypi.org/project/pulumiverse_time/)
-* Go: [`github.com/pulumiverse/pulumi-time/sdk/go/time`](https://github.com/pulumiverse/pulumi-time/sdk/go/time)
```

Comment:
@susanev I asked for a new release of the provider: https://github.com/pulumiverse/pulumi-time/issues/24